player/types.go

```diff
@@ -0,0 +1,13 @@
+package player
+
+type RoundParts struct {
```

Comment:
this is a DB structure, we will need a converter (in an ops package) to turn it to and from a player.RoundInfo

Reply:
This is the correct location for the DB struct though right? This is all new to me

Reply:
no we need you to put the db struct into the db/parts package

Reply:
I'm not sure I've understood correctly, but it's now in db/parts/types.go. It feels weird to me, since this struct kinda belongs in both /parts and /rounds